<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSRWindow</class>
 <widget class="QDialog" name="CSRWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSR Assistant</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblDomain">
       <property name="text">
        <string>&amp;Domain</string>
       </property>
       <property name="buddy">
        <cstring>txtDomain</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtDomain">
       <property name="text">
        <string>www.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblCountry">
       <property name="text">
        <string>Country</string>
       </property>
       <property name="buddy">
        <cstring>txtCountry</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtCountry">
       <property name="text">
        <string>IT</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblStateProvince">
       <property name="text">
        <string>S&amp;tate or Province</string>
       </property>
       <property name="buddy">
        <cstring>txtStateProvince</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtStateProvince"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblLocality">
       <property name="text">
        <string>&amp;Locality</string>
       </property>
       <property name="buddy">
        <cstring>txtLocality</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtLocality"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblOrganization">
       <property name="text">
        <string>Or&amp;ganization</string>
       </property>
       <property name="buddy">
        <cstring>txtOrganization</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="txtOrganization"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblOrganizationUnit">
       <property name="text">
        <string>Organization &amp;Unit</string>
       </property>
       <property name="buddy">
        <cstring>txtOrganizationUnit</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="txtOrganizationUnit">
       <property name="text">
        <string>Web Department</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblEmail">
       <property name="text">
        <string>Email</string>
       </property>
       <property name="buddy">
        <cstring>txtEmail</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="txtEmail"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkSign">
     <property name="text">
      <string>Generate self-signed certificate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupSign">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Sign options</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblDays">
        <property name="text">
         <string>Days of validation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinDays">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="value">
         <number>365</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblHostKey">
     <property name="text">
      <string>Private host key:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblCSR">
     <property name="text">
      <string>Certificate request:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>236</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnGenerate">
       <property name="text">
        <string>&amp;Generate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtDomain</tabstop>
  <tabstop>txtCountry</tabstop>
  <tabstop>txtStateProvince</tabstop>
  <tabstop>txtLocality</tabstop>
  <tabstop>txtOrganization</tabstop>
  <tabstop>txtOrganizationUnit</tabstop>
  <tabstop>txtEmail</tabstop>
  <tabstop>chkSign</tabstop>
  <tabstop>spinDays</tabstop>
  <tabstop>btnGenerate</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chkSign</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupSign</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
